app/ide-desktop/common/package.json

```diff
@@ -28,14 +28,14 @@
     "test": "vitest run"
   },
   "peerDependencies": {
-    "@tanstack/query-core": "5.45.0",
-    "@tanstack/vue-query": ">= 5.45.0 < 5.46.0"
+    "@tanstack/query-core": "5.54.1",
+    "@tanstack/vue-query": ">= 5.54.0 < 5.56.0"
```

Comment:
With this update, `vueQuery.ts` in `enso-common` is no longer needed--the fix it implements has been merged upstream, so we can use the standard `vueQuery.QueryClient`.